ml/git-log-to-jsonl

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import git
```

Comment:
```suggestion
try:
    import git
except ModuleNotFoundError:
    print("please install GitPython to use this script")
    sys.exit(1)
```

Or just throw in a note `# requries GitPython` so they'll know which dep they need :+1: